rust/macros/module.rs

```diff
@@ -355,7 +355,12 @@ impl ModuleInfo {
                 "params" => info.params = Some(expect_group(it)),
                 _ => panic!(
                     "Unknown key \"{}\". Valid keys are: {:?}.",
-                    key, EXPECTED_KEYS
+                    key,
+                    ALL_KEYS
+                        .iter()
+                        .map(|e| e.1.to_string())
```

Comment:
There is no need to call `to_string()` here. You can join `Vec<&'static str>` as well. After joining the result is `String`, so you should use `{}` instead of `{:?}`.

Reply:
Done.
Thanks for comment. :)